chia/daemon/client.py

```diff
@@ -34,7 +34,11 @@ async def listener():
                 except websockets.exceptions.ConnectionClosedOK:
                     return None
                 decoded = json.loads(message)
-                id = decoded["request_id"]
+                try:
+                    id = bytes32.from_hexstr(decoded["request_id"])
+                except (AttributeError, TypeError, ValueError):
+                    # treat an invalid request id the same as one we are unaware of
+                    continue
```

Comment:
```suggestion
                    pass
```

Using `continue` would just drop this request, which is a new behaviour.